compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -4442,7 +4442,7 @@ declare_lint! {
 
 declare_lint! {
     /// The `refining_impl_trait_reachable` lint detects `impl Trait` return
-    /// types in method signatures that are refined by a publically reachable
```

Comment:
"publically" is less common than "publicly", but does get some use

Reply:
fixed